src/unix/fs.c

```diff
@@ -824,6 +824,16 @@ static ssize_t uv__fs_copyfile(uv_fs_t* req) {
     goto out;
   }
 
+#ifdef FICLONE
```

Comment:
FICLONE isn't visible unless you `#include <linux/fs.h>` - and even then it won't be in many mainstream distros' /usr/include for a while.